File: src/test/java/test/GetUserLoggedInIdentitiesTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetUserLoggedInIdentitiesTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetUserLoggedInIdentitiesDto request = new GetUserLoggedInIdentitiesDto();
        request.setUserId("userId_2279");

        UserLoggedInIdentitiesRespDto response = client.getUserLoggedInIdentities(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetUserLoggedinIdentitiesTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetUserLoggedInIdentitiesDto{
        UserId: "userId_2447",
        UserIdType: "userIdType_2827",
    }
    response := client.GetUserLoggedinIdentities(&request)
    fmt.Println(response)
}

